utils/stats.cpp

```diff
@@ -56,6 +57,11 @@ template <typename T> vector<T> Stats<T>::get_vals() {
     return m_vals;
 }
 
+template <typename T> void Stats<T>::reset_vals() {
+    m_vals.clear();
+}
+
 // Explicit template instantiation
 template class Stats<double>;
-template class Stats<int>;
\ No newline at end of file
+template class Stats<int>;
+template class Stats<uint64_t>;
```

Comment:
not sure if we need this, in most cases this will be a double since we divide by number of iterations

Reply:
Removed in the next commit.